javascript-source/commands/customCommands.js

```diff
@@ -135,11 +135,13 @@
         }
 
         if (message.match(/\(1=[^)]+\)/g)) {
+            var t = message.match(/\(1=[^)]+\)/)[0];
```

Comment:
This function is made to accept command variables, not random text. It it working fine.

Reply:
But if I understand correctly, there is (1) only for variables, and (1=default) here, if you want to see something by default to be written by the bot in chat, without brackets. As when you use variables, there are no brackets around them in text message.

Reply:
There's a bunch of variables, this meant to be used like this. `(1=random)`, if the first argument isn't passed, it will use the `(random)` variable.

Reply:
Ah, I see now...